src/rgw/rgw_rados.h

```diff
@@ -2595,10 +2582,6 @@ class RGWRados
 
   int create_pool(const rgw_pool& pool);
 
-  /**
-   * create a bucket with name bucket and the given list of attrs
-   * returns 0 on success, -ERR# otherwise.
-   */
   int init_bucket_index(RGWBucketInfo& bucket_info, int num_shards);
```

Comment:
@mikulely Yeah, you are right. If you think dropping it then drop the same in the implementation too `rgw_rados.cc`; or update it properly as per the new prototype. If I was doing it, I would prefer updating it as per the latest prototype.